configure.ac

```diff
@@ -1193,6 +1193,7 @@ AC_SUBST(QR_LIBS)
 AC_CONFIG_FILES([Makefile src/Makefile share/setup.nsi share/qt/Info.plist])
 AC_CONFIG_FILES([contrib/devtools/split-debug.sh],[chmod +x contrib/devtools/split-debug.sh])
 AM_COND_IF([HAVE_DOXYGEN], [AC_CONFIG_FILES([doc/Doxyfile])])
+AC_CONFIG_LINKS([CHANGELOG.md:CHANGELOG.md COPYING:COPYING])
```

Comment:
`share/setup.nsi.in` fails to resolve this on GA for some reason, will investigate further.